languages/java11/template/src/main/java/com/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,26 @@
+package com.algorithmia.algorithm;
+
+import com.algorithmia.development.*;
+import com.algorithmia.*;
+
+
+// This class defines your algorithm.
+class Algorithm implements AlgorithmInterface<String, String>{
+
+
+    public String apply(String input){
```

Comment:
As a first time user, I feel like it's not clear that the input type and output type here needs to match the type parameters of AlgorithmInterface

Reply:
If they don't match they'll get an IDE warning, but we could add a comment here to make that more explicit.

Reply:
They won't get a IDE warning in the web IDE though. I think adding a comment makes sense.